src/Mod/Part/App/TopoShapeExpansion.cpp

```diff
@@ -757,9 +757,13 @@ void TopoShape::mapSubElementTypeForShape(const TopoShape& other,
                 }
             }
             char elementType {shapeName(type)[0]};
+
+            // Originally in ComplexGeoData::setElementName
+            // LinkStable/src/App/ComplexGeoData.cpp#L1631
```

Comment:
Note to self: add the same comment of `ElementMap::setElementName`, here and on the modifications below